root/admin/EmailSearch.js

```diff
@@ -42,6 +42,15 @@ const EmailSearch = ({
             },
           )}
         </p>
+        <p>
+          {exp.l(
+            `Don’t forget to escape <code>\\.</code> and <code>\\+</code>!
+             And keep in mind the search ignores periods (<code>.</code>)
+             and any sections after <code>+</code> in the user side of the
+             address (before the <code>@</code>)
+             to help you find address aliases.`,
```

Comment:
* Only one backslash (`\`) is needed to escape special characters in search input field.
* “Keep in mind” probably refers to the comment in the Perl code, or to the ticket. Maybe better explains it once instead.
* It “ignores periods (`.`)” from the `email` column, but it ignores escaped periods (`\.`) from the input.
* Same for tags following an (escaped) plus symbol.

I tried to rephrase it as follows to fix the above issues. I did neither test nor lint it.

```suggestion
            `Since periods (<code>.<code>) and tags preceded with a <code>+</code> sign
             on the user side of the address (that is before the <code>@</code> sign)
             are often used as “free aliases” by email providers, both of these are ignored
             by this search to help you find address aliases.<br/>
             Don’t forget to escape <code>\.</code> and <code>\+</code>
             still as they will be considered as special characters otherwise!`,
```

Reply:
> Only one backslash (\) is needed to escape special characters in search input field.

Yeah, but you need two here so that one is printed, otherwise it's interpreted as an escape character in the first place.


> “Keep in mind” probably refers to the comment in the Perl code, or to the ticket. Maybe better explains it once instead.

The idea was that we expect any admin would be aware of this in theory, but fine, I guess it doesn't hurt to change the way it's explained :)

Reply:
Updated the text, see how it looks to you :)

Reply:
Good catch for the double backslash, I thought that `\.`would be enough because that is not a valid escape here.